docs/source/basics/latch_develop_example.md

```diff
@@ -0,0 +1,251 @@
+
+$ cd simple-variant-calling
+```
+
+The repository consists of three folders: 
+* `buggy-wf`: The workflow with bugs 
+* `good-wf`: The final, functional workflow
+* `wgs`: Test data for the workflow 
+
+After this example, you will learn how to use `latch develop` to effectively test and debug the `buggy-wf` to arrive at the `good-wf`.
+
+Let's get started!
+
+## Preparing test data 
+First, we have to upload our test data folder, `wgs`, to the Latch Platform. 
+
+To do so, you can navigate to [console.latch.bio](https://console.latch.bio), and drag and drop the test data folder on Latch. You should see a spinning wheel which indicates the status of your data upload. 
+
+![Upload](../assets/latch-develop-example/data-upload.png)
+
+Once your data has finished uploading, you can verify whether it exists on Latch by using `latch ls` like so: 
+```
+$ latch ls
+
+Size Date Modified Name    
+   - -             welcome/
+   - -             wgs/    
+```
+
+## Overview of the variant calling workflow
+The data we are working with is part of a long-term evolution experiment by Richard Lenski. The experiment was designed to assess adaptation of _E. coli_. A population was propgated for more than 40,000 generations in a glucose-limited minimal medium that was supplemented with citrate. Sequencing of the populations at differnt time points revealed that that spontaneous citrate-using variant (Cit+) appeared between 31,000 and 31,500 generations, causing an increase in population size and diversity. 
+
+Variant calling is a common workflow used to see
+how the population of _E. coli_ changed over time relative to the original population, _E. coli_ strain REL606. Therefore, we will align each of our samples to the _E. coli_ strain REL606 reference genome to determine what differences exist between our reads versus the genome. 
+
+![Upload](../assets/latch-develop-example/variant-calling-wf.png)
+
+Our variant calling pipeline consists of five sequential steps: 
+1. Index the reference genome
+2. Align reads to reference genome
+3. Convert SAM to BAM format 
+4. Sort BAM file by coordinates
+5. Variant calling
+
+Correspondingly, the Latch SDK workflow contains five tasks (`_build_index`, `align_reads`, `convert_to_bam`, `sort_bam`, and `variant_calling`).
+
+We've provided the code for the five tasks above for you in the `buggy-wf`, which you will now test and debug! 
+
+## Testing and Debugging the Workflow 
+* First, enter the `buggy-wf` folder and register it to Latch: 
+```
+$ ls 
+buggy-wf        good-wf         wgs
+
+$ cd buggy-wf
+
+$ latch register --remote .
+```
+
+## Defining a test script
+* Before testing the workflow end-to-end, it is helpful to run and test each task individually. To do so, you can create a folder called `scripts`, which contain a Python file that calls each of your task functions. 
+
+```
+# Folder structure inside buggy-wf
+.
+├── Dockerfile
+├── README.md
+├── scripts
+│   └── main.py
```

Comment:
nit but i wouldn't call this `main.py` since this isn't starting an application, would prefer something like `test_tasks.py` or something similar